tensorflow_lite_support/python/task/vision/core/utils.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
As this utils will be used by both vision, text and audio, please move it to `tensorflow_lite_support/python/task/core/utils.py`